src/components/Home/ConnectionCard/ConnectionCard.tsx

```diff
@@ -10,10 +10,12 @@ import LoadingBox from "../../Shared/LoadingBox/LoadingBox";
 
 const HIDDEN_TEXT = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
 
-export const ConnectionCard: FC<ConnectionCardProps> = ({
-  sshAddress,
-  torAddress,
-}) => {
+export type Props = {
```

Comment:
Just for future consistency, should we move type declarations to the top? Just asking because in components it's placed at the bottom. I would vote for top as well :)

Reply:
Yeah saw that with Alby and found that to be  more readable than bottom.

Will migrate step-by-step when touching the component :)